SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -143,19 +139,19 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	UPROPERTY()
 	USpatialStaticComponentView* StaticComponentView;
 	UPROPERTY()
-	USnapshotManager* SnapshotManager;
-	UPROPERTY()
 	USpatialMetrics* SpatialMetrics;
 	UPROPERTY()
 	ASpatialMetricsDisplay* SpatialMetricsDisplay;
 	UPROPERTY()
 	ASpatialDebugger* SpatialDebugger;
 	UPROPERTY()
-	USpatialLoadBalanceEnforcer* LoadBalanceEnforcer;
-	UPROPERTY()
 	UAbstractLBStrategy* LoadBalanceStrategy;
 
-	TUniquePtr<SpatialVirtualWorkerTranslator> VirtualWorkerTranslator;
+	TSharedPtr<SpatialDispatcher> Dispatcher;
```

Comment:
Alex and I has a chat about possibly making these all unique pointers. This is just a reminder that maybe we want to do that?

Reply:
Added to summary below.